pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.jsqlparser</groupId>
     <artifactId>jsqlparser</artifactId>
-    <version>3.1-SNAPSHOT</version>
+    <version>3.1-SNAPSHOT-ActFact-1</version>
```

Comment:
Please don't change the version.

Reply:
How can I fix this? Revert the change and create a new pull request?

Reply:
Just commit into your pull request. I will see the changes immediatly. I do always a Squash and Merge.